src/kvstore/comm.h

```diff
@@ -191,7 +191,7 @@ class CommCPU : public Comm {
   }
 
   void BroadcastRowSparse(int key, const NDArray& src,
-                          const std::vector<std::pair<NDArray*, NDArray>>& dst,
+                          const std::vector<std::pair<NDArray*, NDArray>> dst,
```

Comment:
?

Reply:
Reverted after verifying that passing by reference doesn't cause scope issues